english/let-me-out/let-me-out.md

```diff
@@ -0,0 +1,19 @@
+You take a moment to collect yourself. 
+Who wouldn't after all the craziness that has happened?
+"Doctor, you have to let me out. I'm not crazy."
+The doctor looks at you, skeptically.
+
+"You should know how this works by now," he sighs.
+And continues, "My hands are tied. There's nothing I can do."
+
+You notice that he is hiding something behind his back.
+
+Do you:
+
+[Kick the Doctor](.kicktheman/kicktheman.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.